spec/lib/scripts/collections_tags_and_mesh_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+require "#{Rails.root}/lib/scripts/collections_tags_and_mesh"
+
+RSpec.describe 'lib/scripts/collections_tags_and_mesh.rb' do
+  let(:user) { FactoryGirl.create(:user) }
+
+  subject { CollectionsTagsAndMeshList.new }
+
+  describe 'CollectionsTagsAndMeshList' do
+    before do
+      make_collection(user, id: "123", title: "Tags and Mesh List",
+                      tag: ["CERN"], mesh: ["invenioRDM"])
+    end
+
+    describe '#populate_collections_tags_and_mesh_info' do
+      it 'adds collections mesh and tags info to collections_tags_and_mesh_list.csv' do
+        subject.populate_collections_tags_and_mesh_info
+        collections_tags_and_mesh_list_result = File.readlines(
+          "#{Rails.root}/lib/scripts/results/collections_tags_and_mesh_list.csv"
+        )
+        expected_data = "Tags and Mesh List,"\
+                        "https://digitalhub.northwestern.edu/collections/123,"\
+                        "CERN,invenioRDM\n"
+
+        expect(collections_tags_and_mesh_list_result[1]).to eq(expected_data)
+      end
+    end # #populate_user_info
+  end
+
+  after do
+    FileUtils.rm_rf(Dir["#{Rails.root}/lib/scripts/results/collections_tags_and_mesh_list.csv"])
```

Comment:
I would use [`FileUtils.rm_f`](https://ruby-doc.org/stdlib-2.4.1/libdoc/fileutils/rdoc/FileUtils.html#method-c-rm_f) since it's for single files instead of `FileUtils.rm_rf` which I think is for directories and contents. I'm not sure what the vulnerability mentioned in `FileUtils.rm_rf`'s documentation is, but might be worth considering too.

Reply:
Ah, good idea. I didn't realize the vulnerability, but this is better anyways.

Reply:
To be honest with you, I didn't even read _what_ the vulnerability was lol. I just figured that it was better to 1: avoid whatever the vulnerability was and 2: not leave room for us to potentially remove an entire directory by accident.